erp/jobs/import_centres_vaccination.py

```diff
@@ -0,0 +1,122 @@
+# Imports vaccination centers from open data: https://www.data.gouv.fr/fr/datasets/lieux-de-vaccination-contre-la-covid-19/
+
+from django.conf import settings
+import requests
+import logging
+
+from core import mailer
+from erp.mapper.vaccination import RecordMapper
+from erp.models import Activite
+
+logger = logging.getLogger(__name__)
+
+
+def outputPrintStrategy(text, end="\n", flush=False):
+    print(text, end=end, flush=flush)
+
+
+def outputVoidStrategy(text, end="\n", flush=False):
+    pass
```

Comment:
That would rather be `logger.info(text)`, and if we add a `level` arg to this interface we could even log messages at a given level (for the records, `error` will be caught by Sentry, `info` won't).